### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by gregdenay** [bio-raum/FooDMe2#15] [General]
@marchoeppner 

I could get started on creating a github-page for the docs. If you don't have a favorite framework I would suggest materials for mkdoc, that's what I used for [FooDMe](https://cvua-rrw.github.io/FooDMe/) and [taxidTools](https://cvua-rrw.github.io/taxidtools/), it is quite easy, looks nice, and everything can be tweaked.

**Comment by marchoeppner** [comment]
Haven't used github pages or readthedocs yet, so am completely open to any and all options. I'll leave that choice to you and will complain afterwards ;)

**Comment by gregdenay** [comment]
Deal :p

**Comment by gregdenay** [comment]
https://bio-raum.github.io/FooDMe2/
